src/routes/(console)/project-[project]/storage/bucket-[bucket]/usage/[[period]]/+page.svelte

```diff
@@ -5,16 +5,30 @@
     import type { PageData } from './$types';
 
     export let data: PageData;
-    $: total = data.filesTotal;
-    $: count = data.files;
+    $: filesTotal = data.filesTotal;
+    $: files = data.files;
+    $: transformationsTotal = data.imageTransformationsTotal;
+    $: transformations = data.imageTransformations;
+
+    console.log(data);
 </script>
 
 <Usage
     title="Files"
     path={`${base}/project-${$page.params.project}/storage/bucket-${$page.params.bucket}/usage`}
-    {total}
-    {count}
+    total={filesTotal}
+    count={files}
     countMetadata={{
         legend: 'Files',
         title: 'Total files'
     }} />
+
+<Usage
+    title="Image Transformations"
+    path={`${base}/project-${$page.params.project}/storage/bucket-${$page.params.bucket}/usage`}
+    total={transformationsTotal}
+    count={transformations}
+    countMetadata={{
+        legend: 'Image Transformations',
+        title: 'Total transformations'
+    }} />
```

Comment:
we might have to change this component maybe to optionally hide the period selector. There should only be on per usage page, currently it shows 2 of these -

![image](https://github.com/user-attachments/assets/4dd6ceb8-3cd7-4e61-be69-89ed8b71c324)